mappings/net/minecraft/util/DyeColor.mapping

```diff
@@ -18,6 +18,9 @@ CLASS net/minecraft/class_1767 net/minecraft/util/DyeColor
 	METHOD method_7786 byFireworkColor (I)Lnet/minecraft/class_1767;
 		ARG 0 color
 	METHOD method_7787 getColorComponents ()[F
+		COMMENT Returns the red, blue and green components of this dye color.
```

Comment:
Wait, in that order?

Reply:
Yep, I checked with some dye color values